modules/crafting/client.lua

```diff
@@ -94,6 +94,10 @@ local function createCraftingBench(id, data)
 	end
 end
 
-for id, data in pairs(lib.load('data.crafting')) do createCraftingBench(id, data) end
+RegisterNetEvent('ox_inventory:registerCraftingBench', function(id, data)
+    createCraftingBench(id, data)
+end)
+
+lib.callback('ox_inventory:getCraftingBenches', false, function(data) for id, data in pairs(data) do createCraftingBench(id, data) end end)
```

Comment:
I would rename the var `data` in the loop
```suggestion
lib.callback('ox_inventory:getCraftingBenches', false, function(data) for id, bench in pairs(data) do createCraftingBench(id, bench) end end)
```